static/src/javascripts/projects/common/modules/ui/subscription-medium-banner.js

```diff
@@ -0,0 +1,242 @@
+// @flow
+
+import uniq from 'lodash/uniq';
+import { hasUserAcknowledgedBanner } from 'common/modules/ui/message';
+// import { trackNonClickInteraction } from 'common/modules/analytics/google';
+import config from 'lib/config';
+import userPrefs from 'common/modules/user-prefs';
+import type { Banner } from 'common/modules/ui/bannerPicker';
+import { local } from 'lib/storage';
+import {
+    submitViewEvent,
+    submitClickEvent,
+} from 'common/modules/commercial/acquisitions-ophan';
+import marque36icon from 'svgs/icon/marque-36.svg';
+// import {  getSync as geolocationGetSync } from 'lib/geolocation';
+import {
+    track as trackFirstPvConsent,
+    // canShow as canShowFirstPvConsent,
+    // messageCode as firstPvConsentMessageCode,
```

Comment:
We probably still want to import these and use the consent messaging modules rather than adding the html here.
If the firstPvConsentMessageCode changes, then it might mean that this design drifts from the one intended by CommDev.